LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/IT87XX.cs

```diff
@@ -518,7 +518,7 @@ private void RestoreDefaultFanPwmControl(int index)
 
         private readonly byte[] FAN_PWM_CTRL_REG;
         private readonly byte[] FAN_PWM_CTRL_EXT_REG = { 0x63, 0x6b, 0x73, 0x7b, 0xa3 };
-        private readonly byte[] FAN_TACHOMETER_EXT_REG = { 0x18, 0x19, 0x1a, 0x81, 0x83, 0x4c };
+        private readonly byte[] FAN_TACHOMETER_EXT_REG = { 0x18, 0x19, 0x1a, 0x81, 0x83, 0x4d };
```

Comment:
What's the source of this?

Reply:
Main and extended register has same address for fan5.
https://github.com/torvalds/linux/blob/0513e464f9007b70b96740271a948ca5ab6e7dd7/drivers/hwmon/it87.c#L226

Reply:
Thanks 👍